### ISSUE TITLE
Update docs, pypi, etc. to state support for Python 2.7

### ISSUE THREAD

**Issue by mozbhearsum** [zalando/python-clickclick#11]
Everything seems to claim this library is Python3 only, eg: https://pypi.python.org/pypi/clickclick/1.2.1

**Comment by hjacobs** [comment]
You are right, the docs etc were not updated together with #9. Thanks for taking the time to create an issue :smile:
